### PR TITLE
Miro connector

### PR DESCRIPTION
**Pull request by ArtemKrivtsov**

### CODE REVIEW

File: custom_connectors/oauth2/miro.rb [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    }
+  },
+
+  actions: {
+
+    create_board: {
+      description: 'Creates a Board in Miro',
+      input_fields: lambda do
+        [
+          { name: 'name', label: 'Title' },
```
[the remark this answers is not in the document]
Comment (sharat-developer) [reply custom_connectors/oauth2/miro.rb]: Also, consider using [`sticky`](https://docs.workato.com/developing-connectors/sdk/object-definition.html#components) attribute for the prominent fields of the object, to make the fields show-up for the user - without having to choose from `Add or remove optional fields` button in the recipe.
Ref: https://github.com/workato/custom_connector_docs/blob/master/custom_connectors/api_key_auth/clover.rb

File: custom_connectors/oauth2/miro.rb [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    end,
+
+    boards: lambda do
+      account_id = get('/v1/oauth-token')['account']['id']
+      query = 'fields=id,name&limit=500'
+      boards_resp = get("v1/accounts/#{account_id}/boards?#{query}")
+      next_link = boards_resp['nextLink']
+      boards = boards_resp['data']
+
+      while next_link.present?
```
Comment (bennettgo) [inline custom_connectors/oauth2/miro.rb]: How many boards do 80% of your users have? thats normally a good benchmark for how many boards to return in a picklist

Comment (bennettgo) [reply custom_connectors/oauth2/miro.rb]: We normally don't encourage loops when building out picklists

**Comment by ArtemKrivtsov** [comment]
closed because pull request was detached from the repository

**Pull request closed without merging**